lib/components/search_user/interactor/search_user_bloc.dart

```diff
@@ -39,8 +40,12 @@ class SearchUserBloc extends Bloc<SearchUserEvent, SearchUserState> {
 
       if (event.searchQuery.length > _minTextLengthBeforeValidSearch) {
         yield state.copyWith(pageState: PageState.loading);
-        final result = await SearchForMemberUseCase().run(event.searchQuery);
-        yield SearchUserStateMapper().mapResultToState(state, result, state.noShowUsers);
+        final futures = [
```

Comment:
this should be done in the use case. 

I dont think you need a use case for telos and one for seeds. 

You need one use case of members. And inside the use case you call both telos and seeds repo calls

Reply:
Done